config/index.js

```diff
@@ -52,7 +50,7 @@ log('>>>>', configs);
 _.extend(Matrix, configs);
 
 
-f.jwt = { secret : process.env.JWT_SECRET }
+f.jwt = { secret : process.env.JWT_PUBLIC_KEY }
```

Comment:
This can be used to validate tokens if needed, but didn't find where in the code this is currently used.